androidClient/app/src/main/java/com/macbitsgoa/comrades/courselistfragment/CourseListFragment.java

```diff
@@ -107,6 +107,9 @@ public View onCreateView(@NonNull final LayoutInflater inflater,
             arrayList.addAll(courses);
             courseAdapter.notifyDataSetChanged();
         });
+        courseVm.courseCount.observe(this,
```

Comment:
@Rushi98  Instead of using  special function for course count, we can just do `TextView.setText(list.size());` there.

Reply:
I'm planning to use paging for the recycler views to make the UI faster. After which we won't be able to use list.size to get total count. So, count and list are made independent.